### PR TITLE
UI/ux

### PR DESCRIPTION
**Pull request by Miminiverse**
This PR provides:
1/ Sort users list by latest contact
2/ Add language selection for users, and the messages will be translated into language that log-in user set up  
3/ **Fix bugs**Whenever click in non-contact people, will render a completely new chat container
2/ An update on UI/UX and a draft for final main

### CODE REVIEW

File: src/components/ChatContainer.tsx [hunk 1]
```diff
@@ -45,9 +47,10 @@ const ChatContainer = ({ socket }: { socket: Socket }): JSX.Element => {
                     }
                   }
                 )
-                
+                console.log(data)
```
Comment (royemosby) [inline src/components/ChatContainer.tsx:50]: Don't forget to remove console.logs after debugging

File: src/components/ChatContainer.tsx [hunk 2]
```diff
@@ -62,25 +65,38 @@ const ChatContainer = ({ socket }: { socket: Socket }): JSX.Element => {
...
+        console.log(data);
+        setSelectedTyping(data)
         setIsTyping(true)
       }
       );
       socket.current.on("stopTyping", () => setIsTyping(false));
     }
   }, [socket.current]);
 
+  console.log(selectedTyping)
```
Comment (royemosby) [inline src/components/ChatContainer.tsx:86]: ^^

File: src/components/ChatContainer.tsx [hunk 3]
```diff
@@ -102,47 +118,90 @@ const ChatContainer = ({ socket }: { socket: Socket }): JSX.Element => {
...
+                 message: messageText
+             },
+             {
+                 headers: {
+                   Authorization: `Bearer ${token}`
+                 }
+               }
+             )
+             const {message} = data
+             console.log(data)
```
Comment (royemosby) [inline src/components/ChatContainer.tsx:173]: ^^

**Review by royemosby — APPROVED** [review]
Looks good! don't forget to remove extraneous `console.log`s as well as unused, commented out code for your MVP

File: src/components/ChatContainer.tsx (hunk 3 above)
Comment (royemosby) [reply src/components/ChatContainer.tsx:173]: There are a few others but I don't want to clutter the PR thread

Comment (Miminiverse) [reply src/components/ChatContainer.tsx]: Hi Roy 
Got it and will update!

**Review by Said-laasri — APPROVED** [review]
Hey @Miminiverse  :+1:  Great Job :100: 

Your PR is good to go && Ready for the merge.



HAPPY CODING!! :1st_place_medal:

**Review by royemosby — APPROVED** [review]
🌮 GTG!

**Pull request merged**